Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -106,7 +111,8 @@ protected override void OnDetachedFromWindow()
 			base.OnDetachedFromWindow();
 
 			_isAttached = false;
-			_adapter.IsAttachedToWindow = _isAttached;
+			if (_adapter != null)
```

Comment:
If this object is resurrected via the JniHandleOwnership ctor for purposes of allowing Android to call `Dispose`, then `_adapter` is null.